drivers/audio/Kconfig

```diff
@@ -21,7 +21,7 @@ if AUDIO_CODEC
 
 config AUDIO_CODEC_INIT_PRIORITY
 	int "Init priority"
-	default 60
+	default 90
```

Comment:
Don't sneak small but super generic and impactful changes like this in such a huge, mostly hardware-specific PR. Also, this change is not even explained by a commit message (because nothing has a commit message right now)

https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#defining-smaller-prs
https://github.com/zephyrproject-rtos/zephyr/pull/65558#discussion_r1449355733

(We had plenty of subtle and painful build failures caused by init priorities in the past)

cc: @fabiobaltieri

Reply:
> We had plenty of subtle and painful build failures caused by init priorities in the past

Yeah well, writing that script and fixing the whole upstream code base to make it pass was plenty unsubtly painful too. :-)